package.json

```diff
@@ -196,14 +197,17 @@
     "@emotion/styled": "^11.8.1",
     "@fontsource/open-sans": "^4.5.10",
     "@react-nano/use-event-source": "^0.12.0",
+    "bezier-js": "^6.1.0",
     "cross-fetch": "^3.1.5",
     "decompress": "^4.2.1",
     "electron-log": "^4.4.8",
     "electron-squirrel-startup": "^1.0.0",
     "fast-deep-equal": "^3.1.3",
     "framer-motion": "^6.3.6",
+    "install": "^0.13.0",
```

Comment:
This is a reminder to myself to uninstall these, i must have done "npm install npm install" lol